lxc/init.go

```diff
@@ -281,47 +282,16 @@ func (c *cmdInit) create(conf *config.Config, args []string) (lxd.InstanceServer
 	// If there are device overrides that are expected to be applied to profile devices then load the profiles
 	// that would be applied server-side.
 	if needProfileExpansion {
-		// If the list of profiles is empty then LXD would apply the default profile on the server side.
```

Comment:
Don't we want to keep this comment to keep the information on what is happening in case the list of devices is empty?

Reply:
I added this information above getProfileDevices but included it here also